src/StaticWebAssetsSdk/Tasks/DiscoverStaticWebAssets.cs

```diff
@@ -17,7 +17,7 @@ public class DiscoverStaticWebAssets : Task
         public ITaskItem[] Candidates { get; set; }
 
         [Required]
-        public string Pattern { get; set; }
+        public ITaskItem[] Patterns { get; set; }
```

Comment:
Can we avoid taking multiple patterns here? This can be called via MSBuild batching from outside instead, and simplifies things.